packages/core/src/components/html-select/_html-select.scss

```diff
@@ -67,18 +67,3 @@
     }
   }
 }
-
-.#{$ns}-html-select {
-  .#{$ns}-icon {
-    @extend %pt-select-icon;
-  }
-}
```

Comment:
Removed this also since it duplicates the computed selector above: https://github.com/palantir/blueprint/blob/c09fa6a9956eaed242056c9ff755b5159794c8f0/packages/core/src/components/html-select/_html-select.scss#L23-L26